static/app/utils/api/useAggregatedQueryKeys.tsx

```diff
@@ -90,6 +92,14 @@ export default function useAggregatedQueryKeys<AggregatableQueryKey, Data>({
 
   const key = getQueryKey([]).at(0);
 
+  const subscriptions = useRef<Subscription[]>([]);
+  useEffect(() => {
+    const subs = subscriptions.current;
+    return () => {
+      subs.forEach(unsubscribe => unsubscribe());
```

Comment:
Since we unsubscribe on unmount, seems the cache keys can be evicted if you go to another page (which we probably want). Are there any possible bugs in this scenario or should this hook handle it gracefully?

Reply:
I think the hook will handle it fine, if the keys are evicted then we'll re-fetch. 

Also, because it's a buffer, we could often be re-fetching a different list of id's. For example, you're on the issue stream and have issues `abc-3, abc-2, abc-1` captured so far. We'll fetch `/issue-count/?id=3,2,1`. If you navigate away then back a new issue `abc-4` might've appeared, we'll either fetch only `?id=4` if the cache is still full, or `?id=4,3,2,1` and get new counts for each.